Formula/ibex.rb

```diff
@@ -12,37 +22,59 @@ class Ibex < Formula
     sha256 "41327b6a0da9b8b2ed888c3c2c33d5f6ff061ea13eceb4b9e8b6e660049d624d" => :mountain_lion
   end
 
-  option "with-java", "Build Java bindings for Choco solver."
+  option "with-java", "Enable Java bindings for CHOCO solver."
+  option "with-ampl", "Use AMPL file loader plugin"
+
+  option "without-ensta-robotics", "Don't build the Contractors for robotics (SLAM) plugin"
+  option "without-param-estim", "Don't build the Parameter Estimation (enhanced Q-intersection algorithm) plugin"
 
   depends_on :java => ["1.8+", :optional]
   depends_on "bison" => :build
   depends_on "flex" => :build
   depends_on "pkg-config" => :build
 
   def install
-    args = ["--prefix=#{prefix}"]
-
-    if build.with? "java"
-      args << "--with-jni"
-    else
-      args << "--enable-shared"
+    if build.with?("java") && build.with?("ampl")
+      odie "Cannot set options --with-java and --with-ampl simultaneously for now."
     end
 
+    args = ["--prefix=#{prefix}", "--enable-shared", "--with-affine", "--with-optim"]
+
+    args << "--with-jni" if build.with? "java"
+    args << "--with-ampl" if build.with? "ampl"
+    args << "--with-ensta-robotics" if build.with? "ensta-robotics"
+    args << "--with-param-estim" if build.with? "param-estim"
+
     system "./waf", "configure", *args
     system "./waf", "install"
 
-    cd "examples" do
-      cxxflags = "-frounding-math -ffloat-store -I#{include} -I#{include}/ibex"
-      libflags = "-L#{lib} -libex -lprim -lClp -lCoinUtils -lm"
-      system "make", "defaultsolver", "LIBS=#{libflags}", "CXXFLAGS=#{cxxflags}"
-    end
-
     pkgshare.install %w[examples benchs]
+    (pkgshare/"examples/symb01.txt").write "function f(x)\nreturn ((2*x,-x);(-x,3*x));\nend"
+  end
+
+  def post_install
+    # Build a few examples
+    cd "#{pkgshare}/examples" do
+      # Build Ibex examples
+      system "make", *%w[ctc01 ctc02 symb01 solver01 solver02]
+      # Build SLAM examples
+      cd "slam" do
+        system "make", *%w[slam1 slam2 slam3]
+      end
+    end
```

Comment:
Why are you building stuff in `post_install`? If this is supposed to be part of the software (and distributed with the binaries), it needs to go into `install`. `post_install` is just for lightweight things like adjusting a configuration, something that can only be done once the package is installed on a user's system.

Aside from the above reason, you can't expect to have things available here you have had during install. That's why the log on our build bot shows an error with a missing `pkg-config`.